python/GafferSceneUI/ShaderQueryUI.py

```diff
@@ -263,14 +346,15 @@ def __getLabel( plug, parentPlug = None, sanitise = False ) :
 			if the parameter does not exist.
 			""",
 
+			"label", "value",
```

Comment:
I think we could remove this one too.

Reply:
Fixed in ee772461990f735e576ffd9303f6960ce8ecbe8e